### PR TITLE
Fix various CSS things.

### PR DESCRIPTION
**Pull request by jimmyfnugent**
A number of changes worth noting:

* We're using React-Bootstrap Sass
* Updated to Bootstrap v4.5, which broke a lot of things visually
* Migrated our remaining old CSS class references to Sass, as they weren't working anyway
* If Bootstrap's DropdownButton isn't rendered as a ButtonGroup, apparently it just wraps itself in a div that takes the entire screen width, so you can't put anything on the same horizontal level. Awesome.

Before: 
image

After: 
image

### CODE REVIEW

File: client/src/App.sass [hunk 1]
```diff
@@ -10,6 +12,11 @@
   height: 150px
   padding: 20px
   color: white
+  // These seem to be needed to get the header to be full-width.
+  // See https://css-tricks.com/the-inside-problem/.
+  width: 100vw;
+  margin-left: 50%;
+  transform: translateX(-50%);
```
Comment (hopskipnfall) [inline client/src/App.sass:19]: did running prettier not remove these semicolons?

File: client/src/BitTagPills.tsx [hunk 2]
```diff
@@ -22,11 +23,11 @@ const BitTagPills: AppFunctionComponent<InputProps, NOOP> = (props) => {
     thunkSetLabels,
   } = props;
   return (
-    <div className="bit-tag-pills">
+    <div className={styles['bit-tag-pills']}>
```
Comment (hopskipnfall) [inline client/src/BitTagPills.tsx:26]: i'm not sure what the best practice is here, but in the sass if we name the class bitTagPills i think we might just be able to say `styles.bitTagPills` and have compile time checking? fine with leaving as-is too. there's probably a better way to do this idk.

File: client/src/PageSizeMenu.tsx [hunk 3]
```diff
@@ -18,7 +18,12 @@ const PageSizeMenu: AppFunctionComponent<InputProps, typeof mapStateToProps> = (
   return (
     <span>
       Results per page:
-      <DropdownButton variant="info" title={`${pageSize}`} id="page-size-menu">
+      <DropdownButton
+        as={ButtonGroup}
+        variant="info"
+        title={`${pageSize}`}
```
Comment (hopskipnfall) [inline client/src/PageSizeMenu.tsx:24]: maybe i shouldn't ask but is this a stupid react thing? you can't just write `title={pageSize}`？

**Review by hopskipnfall — APPROVED** [review]

File: client/src/App.sass (hunk 1 above)
Comment (jimmyfnugent) [reply client/src/App.sass:19]: Apparently not -- removed.

File: client/src/BitTagPills.tsx (hunk 2 above)
Comment (jimmyfnugent) [reply client/src/BitTagPills.tsx:26]: I think that is basically the choice AFAICT. Probably the camelcase is better for type checking. Too lazy to fix this now so I'll follow up.

File: client/src/PageSizeMenu.tsx (hunk 3 above)
Comment (jimmyfnugent) [reply client/src/PageSizeMenu.tsx:24]: Looks like it doesn't auto-cast to a string in that case.

**Pull request merged**